bears/documentation/DocumentationStyleBear.py

```diff
@@ -97,6 +99,13 @@ def process_documentation(self,
 
             new_metadata.append(m._replace(desc=new_desc.lstrip(' ')))
 
+        # Check for raw doctrings in python
+        if re.match(r'\s*r', doc_comment.indent):
+            doc_comment.marker = ('r"""', '', '"""')
+            doc_comment.indent = ' ' * (len(doc_comment.indent)-1)
+        else:
+            doc_comment.indent = ' ' * len(doc_comment.indent)
+
```

Comment:
@kriti21 , Is this solution working locally?

Reply:
Yes absolutely! @sks444

Reply:
I tested it locally, It is showing the same patches as previous

Reply:
Can you share the file you are testing it on? I will test it on that and get back to you. I tested these changes locally. I am pretty sure they should work.

Reply:
@kriti21 , Sorry, my bad. I pasted your code at the wrong place.
It is absolutely working fine :+1: